bin/src/command/upgrade.rs

```diff
@@ -249,11 +250,12 @@ impl Gatherer for UpgradeWorkerTask {
                     }
                 }
             }
-            ResponseStatus::Failure => self.errors += 1,
-            ResponseStatus::Processing => client.return_processing(format!(
+            Ok(ResponseStatus::Failure) => self.errors += 1,
+            Ok(ResponseStatus::Processing) => client.return_processing(format!(
                 "Worker {} is processing {}. {}",
                 worker_id, message.id, message.message
             )),
+            _ => {}
```

Comment:
Same here, what is the behavior?

Reply:
this is so unlikely I don't think we need to do anything

Reply:
as unlikely as it is, I'll count it as an error, together with ResponseStatus::Failure:

```
            Ok(ResponseStatus::Failure) | _ => self.errors += 1,
```